Content.Server/Disposal/Unit/EntitySystems/DisposalUnitSystem.cs

```diff
@@ -9,15 +10,20 @@ public override void Initialize()
         {
             base.Initialize();
 
-            SubscribeLocalEvent<DisposalUnitComponent, PhysicsBodyTypeChangedEvent>(BodyTypeChanged);
+
```

Comment:
Remove extra newline

Reply:
i think i've done that one too, this time without removing the ; while im at it...
hopefully